tests/scripts/run-xunit-perf.py

```diff
@@ -149,7 +149,7 @@ def run_command(runArgs, environment, errorMessage):
         subprocess.check_output(runArgs, stderr=subprocess.PIPE, env=environment)
     except subprocess.CalledProcessError as e:
         log(e.output.decode('utf-8'))
-        raise RuntimeException(errorMessage);
+        raise RuntimeError(errorMessage);
```

Comment:
Could you just raise the caught exception?

Reply:
You do not need the `;` at the end of the line

Reply:
Sure.